File: Springboot Code/AgroRent/src/main/java/com/majorproj/agrorent/dto/EquipmentRespDto.java
package com.majorproj.agrorent.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class EquipmentRespDto {
	private Long id;
    private String name;
    private String description;
    private String imageUrl;
    private double rentalPrice;
    private boolean available;
}

File: Dot Net Code/AgroRent/DTOs/EquipmentRespDto.cs
namespace AgroRent.DTOs
{
    public class EquipmentRespDto
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string? ImageUrl { get; set; }
        public double RentalPrice { get; set; }
        public bool Available { get; set; }
        public string? CloudinaryPublicId { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime UpdatedOn { get; set; }
        public int OwnerId { get; set; }
        public string OwnerName { get; set; } = string.Empty;
    }
}
